Agreements/Internal/6_36.txt

```diff
@@ -0,0 +1,33 @@
+COPX:EGLL:*:*:*:*:London S6:London S36:*:30000:^
+COPX:EGWU:*:*:*:*:London S6:London S36:*:30000:^
+
+COPX:EGKK:*:*:*:*:London S6:London S36:*:30000:^
+
+COPX:EGSS:*:*:*:*:London S6:London S36:*:30000:^
```

Comment:
Your notes for ADEP on this say "LTMA (excl. SS)/LF/Wessex/BB/Brize/Severn/Jersey", but you have included EGSS?